Tribute-page.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute for Cristiano Ronaldo</title>
+    <link rel="stylesheet" href="tribute.css">
+</head>
+<body>
+    <div id="main">
+        <h1 id="title">CR7 - The greatest of all time(GOAT)</h1>
```

Comment:
Indeed, CR7 is the Greatest of All Time, my GOAT, your GOAT and this is BECAUSE he plays football exceptionally even better than Messi as it is generally known that practice supersedes talent.🤣
Kindly note that I gave my reasons.

Reply:
thanks, bro